sdk/advisor/azure-mgmt-advisor/azure/mgmt/advisor/_serialization.py

```diff
@@ -271,7 +271,7 @@ def _create_xml_node(tag, prefix=None, ns=None):
         return ET.Element(tag)
 
 
-class Model(object):
+class Model:
```

Comment:
thanks for opening this issue. These are all autogenerated files. I think it is valid to switcvh all of the existing files, I've opened [this](https://github.com/Azure/autorest.python/issues/2942) issue on our generator, so when we regenerate we won't reintroduce these changes